nature.md

```diff
@@ -1,5 +1,5 @@
 # Nature Accessibility Conformance Report International Edition
-(Based on VPAT® Version 2.4Rev)
+(Based on VPAT® Version 2.5Rev)
 
 **Report Date**: 18th August 2023  
```

Comment:
```suggestion
**Report Date**: 24th June 2025  
```

You'll have to update it again soon anyway, once the substantive auditing work starts, but since you're touching the file anyway, it doesn't hurt to update the date for now